patient_portal/lib/doc_lib.php

```diff
@@ -52,12 +50,18 @@
 
 try{
     $form_filename = $_REQUEST['docid'] . '_' . $GLOBALS['pid'] . '.pdf';
-    $templatedir = $GLOBALS['OE_SITE_DIR'] . "/../../patient_portal/patient_documents";
+    $templatedir = $GLOBALS['OE_SITE_DIR'] . "/documents/onsite_portal_documents/patient_documents";
```

Comment:
If these are template directories, the directory should not be named "patient_documents".

Reply:
They are template and the current naming difference will be needed with the Kiosk portion of the application